src/App.js

```diff
@@ -29,17 +29,25 @@ const RouteTitled = ({ title, ...rest }) => {
       document.title = `Civic Tech Index — ${title}`;
     }
   });
-
   return <Route {...rest} />;
 };
 
+
+// conditional render for cookie
+const CookieChecker = () => {
```

Comment:
Since this is a function that conditionally renders a component and not a pure component, we could call it something like `checkCookie` and then below user `{checkCookie}` instead of `<CookieChecker />`

Reply:
Noted, will make changes

Reply:
It should actually be a function call `{checkCookie()}`
Sorry about the confusion